torch/_custom_op/impl.py

```diff
@@ -1078,6 +1078,29 @@ def _find_custom_op(qualname, also_check_torch_library=False):
     return result
 
 
+def get_abstract_impl(qualname):
+    maybe_abstract_impl = get_abstract_impl_from_registry(qualname)
+    if maybe_abstract_impl:
+        return maybe_abstract_impl
+    # No abstract impl registered. Try to see if there's a pystub for it.
+    # This is a bit slow so we do it after the fast-path above
+    did_import = torch._C._dispatch_maybe_import_abstract_impl_pystub(qualname)
+    if did_import:
+        return get_abstract_impl_from_registry(qualname)
```

Comment:
Kind of similar to the meta tensor fallback comment - maybe we could raise a slightly better error message here (than the default "cannot find an abstract impl"), if we know the custom op has an `impl_abstract_pystub("my_py_module")` somewhere, but running it didn't result in an abstract impl populating the table?